app/assets/javascripts/oxalis/api/api_latest.js

```diff
@@ -797,6 +797,7 @@ class UtilsApi {
   *   - MERGE_TREES
   *   - SELECT_NEXT_TREE
   *   - SHUFFLE_TREE_COLOR
+  *   - SHUFFLE_ALL_TREE_COLORS
```

Comment:
Nice! I just noticed I forgot to add quite a few actions here over the time ^.-

Reply:
Maybe forgetting something here is not such a bad thing, since this api function gives away a lot of power, anyway ^^